Calculator.cls.txt

```diff
@@ -0,0 +1,10 @@
+class Series{
+	public static void main(String [] args){
+		int no = 10;
+		int d;
```

Comment:
Use:- int no = 10,d,i; to avoid extra lines.

Reply:
Okay Sir, There is any need to send file again.